src_ts/etools-input/etools-input.ts

```diff
@@ -79,15 +85,26 @@ export class EtoolsInput extends EtoolsInputBase {
             let value = e.target!.value;
 
             if (this.type === 'number') {
-              if (this.min !== '' && this.min !== undefined && value < this.min) {
```

Comment:
Revert this function and use the property allowedPattern directly on the input where is needed